README.md

```diff
@@ -39,12 +50,31 @@ TES Compliant Implementations
 ### Client
 - [cwl-tes](https://github.com/ohsu-comp-bio/cwl-tes)
 - [Cromwell](https://cromwell.readthedocs.io/en/stable/backends/TES/)
-- [Nextflow](https://www.nextflow.io/docs/latest/executor.html#ga4gh-tes)
+- [ELIXIR Cloud Components](https://elixir-cloud-components.vercel.app/)
+- [py-tes](https://github.com/ohsu-comp-bio/py-tes)
+- [Nextflow](https://www.nextflow.io/docs/stable/executor.html#ga4gh-tes)
+- [Snakemake](https://snakemake.readthedocs.io/en/stable/executing/cloud.html#executing-a-snakemake-workflow-via-ga4gh-tes)
+- [Toil](https://toil.readthedocs.io/en/latest/)
+- [St. Jude Rust Labs](https://github.com/stjude-rust-labs/tes)
+
+
 
 ### Server
 - [Funnel](https://ohsu-comp-bio.github.io/funnel/)
 - [TESK](https://github.com/EMBL-EBI-TSI/TESK)
-- [tes-azure](https://github.com/microsoft/tes-azure)
+- [TES Azure](https://github.com/microsoft/ga4gh-tes)
+- [Pulsar](https://pulsar.readthedocs.io/)
+- [proTES](https://github.com/elixir-cloud-aai/proTES)
+
+### Compatibility Matrix
+
+> Compatibility is assumed based on available documentation and limited tests performed on latest versions of implementations available as of December 2020.
+
+|               | cwl-tes       | Cromwell  | Nextflow |
+| -----------   | ------------- | --------- | ---------|
+| **Funnel**    | [Compatible](https://github.com/ohsu-comp-bio/cwl-tes) | Compatible | [Compatible](https://www.nextflow.io/docs/latest/executor.html#ga4gh-tes) |
+| **TESK**      | [Compatible](https://github.com/ohsu-comp-bio/cwl-tes/pull/25) | [Compatible](https://cromwell.readthedocs.io/en/stable/backends/TES/) | [Compatible](https://github.com/EMBL-EBI-TSI/tesk-core/pull/19) |
+| **TES Azure** | Not tested | [Compatible](https://github.com/microsoft/CromwellOnAzure) | [Compatible](https://www.nextflow.io/docs/stable/executor.html#ga4gh-tes) |
 
 
 TES Service Examples
```

Comment:
I think the parts below could use some work:

TES Service Examples section:
- The `name` field in the "Create a task" section includes unncessary whitespace
- Invalid JSON looks bad (highlighted in dark red)
- The `POST` example is incomplete and it isn't clear that the following JSON is a response (rather than, say, a payload)
- I've seen some more typos and smaller issues, probably best to address in a separate issue/PR

Possible Future Enhancements section:
- I would remove this entirely, or rephrase this and add a link to the issue board instad.

Finally, I think it would be nice to include sth like a "Governance" section - which makes clear a bit how we work. People will probably want to know how their proposed changes can make it into the specs (including timelines).